e2e/src/test/java/org/hyades/e2e/BomUploadProcessingE2ET.java

```diff
@@ -225,4 +239,51 @@ private void verifyWebhookNotification() {
         );
     }
 
-}
\ No newline at end of file
+    private void verifyProjectVulnAnalysisCompleteNotification() {
+        wireMock.verify(postRequestedFor(urlPathEqualTo("/notification"))
+                .withRequestBody(equalToJson("""
+                        {
+                           "notification" : {
+                             "level" : "LEVEL_INFORMATIONAL",
+                             "scope" : "SCOPE_PORTFOLIO",
+                             "group" : "GROUP_PROJECT_VULN_ANALYSIS_COMPLETE",
+                             "timestamp" : "${json-unit.any-string}",
+                             "title" : "Project vulnrability analysis complete",
+                             "content" : "${json-unit.any-string}",
+                             "subject" : {
+                               "project" : {
+                                 "uuid": "${json-unit.any-string}",
+                                 "name" : "foo",
+                                 "version" : "bar"
+                               },
+                               "componentAnalysisComplete" : [ {
+                                 "component" : {
+                                   "uuid": "${json-unit.any-string}",
+                                   "group" : "com.fasterxml.jackson.core",
+                                   "name" : "jackson-databind",
+                                   "version" : "2.13.2.2",
+                                   "purl" : "pkg:maven/com.fasterxml.jackson.core/jackson-databind@2.13.2.2?type=jar",
+                                   "md5" : "055c97cb488b0956801e13abcc2a0cfe",
+                                   "sha1" : "ffeb635597d093509f33e1e94274d14be610f933",
+                                   "sha256" : "efb86b148712a838b94b3cfc95769785a116b3461f709b4cc510055a58b804b2",
+                                   "sha512" : "0e9398591d86f80f16fc2d6ff0dda3e7821033e2c59472981eaab61443be3d77198655682905b85260fb2186a2cf0f33988aff689a49bb54e56c07e02f607e8a"
+                                 },
+                                 "vulnerability" : [ {
+                                   "uuid": "${json-unit.any-string}",
+                                   "vulnId" : "INT-123",
+                                   "source" : "INTERNAL",
+                                   "title" : "NA",
+                                   "subTitle" : "NA",
+                                   "recommendation" : "NA",
```

Comment:
Was there a requirement for these fields to always be present? If there is no value, I would've expected the fields to not be present at all.

Reply:
fixed :)